tests/test_form_helper.py

```diff
@@ -487,11 +494,17 @@ def test_bootstrap_form_show_errors_bs5():
 
     form.helper.form_show_errors = True
     html = render_crispy_form(form)
-    assert html.count("error") == 3
+    if django.VERSION < (5, 2):
+        assert html.count("error") == 3
+    else:
+        assert html.count("error") == 6
 
     form.helper.form_show_errors = False
     html = render_crispy_form(form)
-    assert html.count("error") == 0
+    if django.VERSION < (5, 2):
+        assert html.count("error") == 0
```

Comment:
This one concerns me. When this option is enabled Django will add the 'aria_describedby' pointing at a error message that doesn't exist. The docs for this feature say:

> Default set to True. It decides whether to render or not form errors. If set to False, form.errors will not be visible even if they happen. **You have to manually render them customizing your template.** This allows you to customize error output.

The emphasis is mine. Maybe we need to trust that folk doing this know what they are doing? I'm not sure if there's anything else we can do to help here. 

Needs some thought. Opinions welcome.

Reply:
Perhaps pointing to something like [`ErrorList.field_id`](https://docs.djangoproject.com/en/5.2/ref/forms/api/#django.forms.ErrorList.field_id)

> Default set to True. It decides whether to render or not form errors. If set to False, form.errors will not be visible even if they happen. You have to manually render them customizing your template. This allows you to customize error output.
> Note that from Django 5.2, `aria-describedby` will point to the expected error id (`id="{{ errors.field_id }}_error"`) when errors should be rendered. Be sure to have the correct id on your customized template. See [`ErrorList.field_id`](https://docs.djangoproject.com/en/5.2/ref/forms/api/#django.forms.ErrorList.field_id) for details.

Something something.